php/datasource/class-fieldmanager-datasource-term.php

```diff
@@ -76,15 +76,20 @@ public function __construct( $options = array() ) {
 		}
 
 		parent::__construct( $options );
-		if ( $this->only_save_to_taxonomy ) $this->taxonomy_save_to_terms = True;
 
-		// make post_tag and category sortable via term_order, if they're set as taxonomies, and if
-		// we're not using Fieldmanager storage
-		if ( $this->only_save_to_taxonomy && in_array( 'post_tag', $this->get_taxonomies() ) ) {
-			$wp_taxonomies['post_tag']->sort = True;
+		// Ensure that $taxonomy_save_to_terms is true if it needs to be
+		if ( $this->only_save_to_taxonomy ) {
+			$this->taxonomy_save_to_terms = true;
 		}
-		if ( $this->only_save_to_taxonomy && in_array( 'category', $this->get_taxonomies() ) ) {
-			$wp_taxonomies['category']->sort = True;
+
+		if ( $this->taxonomy_save_to_terms ) {
+			// Ensure that the taxonomies are sortable if we're not using FM
+			// storage
+			foreach ( $this->get_taxonomies() as $taxonomy ) {
+				if ( ! empty( $wp_taxonomies[ $taxonomy ] ) ) {
```

Comment:
We're already using `$wp_taxonomies`, so we may as well save the function call IMO